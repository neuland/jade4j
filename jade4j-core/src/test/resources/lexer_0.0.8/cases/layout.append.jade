
extends ../fixtures/append/app-layout

block append head
  script(src='foo.js')
  script(src='bar.js')
