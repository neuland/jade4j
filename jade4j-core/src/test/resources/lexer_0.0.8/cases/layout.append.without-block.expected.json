{"type":"newline","line":2}
{"type":"extends","line":2,"val":"../fixtures/append-without-block/app-layout"}
{"type":"newline","line":4}
{"type":"block","line":4,"val":"head","mode":"append"}
{"type":"indent","line":5,"val":2}
{"type":"tag","line":5,"val":"script","selfClosing":false}
{"type":"attrs","line":5,"attrs":[{"name":"src","val":"'foo.js'","escaped":true}]}
{"type":"newline","line":6}
{"type":"tag","line":6,"val":"script","selfClosing":false}
{"type":"attrs","line":6,"attrs":[{"name":"src","val":"'bar.js'","escaped":true}]}
{"type":"outdent","line":7}
{"type":"eos","line":7}